react/features/speaker-stats/reducer.ts

```diff
@@ -85,31 +90,26 @@ function _updateCriteria(state: ISpeakerStatsState, { criteria }: { criteria: st
  * @returns {Object} - The new state after the reduction of the specified action.
```

Comment:
Can you update the js docs here?

Reply:
I've updated the js docs to remove the comments regarding maintaining the previous order